sdk/src/test/java/com/mapbox/maps/renderer/MapboxRendererTest.kt

```diff
@@ -98,9 +98,16 @@ internal abstract class MapboxRendererTest {
   }
 
   @Test
-  fun setMaximumFpsTest() {
+  fun setValidMaximumFpsTest() {
     mapboxRenderer.setMaximumFps(10)
-    verify { renderThread.setMaximumFps(10) }
+    verify { renderThread.setUserRefreshRate(10) }
+  }
+
+  @Test
+  fun setInvalidMaximumFpsTest() {
+    mapboxRenderer.setMaximumFps(-1)
```

Comment:
it should be two separate tests

Reply:
I'm using verify with 0 counter, imo it's ok to leave as is, don't see a big issue here